frb_example/pure_dart/rust/src/api/misc_no_twin_example_a.rs

```diff
@@ -218,3 +218,15 @@ pub struct StructWithRustAutoOpaqueWithNonCloneData {
 impl StructWithRustAutoOpaqueWithNonCloneData {
     pub fn f(&self) {}
 }
+
+#[cfg(feature = "test")]
+#[allow(non_snake_case)] // There is a bug in spec_generator that requires camel case
+pub fn getFeatureTest() -> String {
+    "test".to_owned()
+}
+
+#[cfg(not(feature = "test"))]
+#[allow(non_snake_case)] // There is a bug in spec_generator that requires camel case
+pub fn getFeatureTest() -> String {
+    "nottest".to_owned()
+}
```

Comment:
issue: hmm, if we have exactly the same function when features are enabled or disabled, then how can we test whether our code generator are using the correct feature? i.e. suppose our code generator does not handle feature flags at all, then it will still output the exact same thing!